core/scss/components/lockup/_lockup--option-r.scss

```diff
@@ -0,0 +1,17 @@
+@charset 'UTF-8';
+
+.su-lockup--option-r {
+
+  .su-lockup__cell2,
+  .su-lockup__cell1 {
+    border: 0;
+  }
+
+  .su-lockup__line1,
+  .su-lockup__line2,
+  .su-lockup__line3,
+  .su-lockup__line4 {
+    display: none;
+  }
+
+}
```

Comment:
From what I recall, and from what I can tell, I don't believe that a school name (e.g., H&S) can be used alone on this line with this option. Something to check in with John H on.

Reply:
Ok, this was from the word doc. If we have to remove it then I am happy to do so. R amount of options is a lot.